src/index.js

```diff
@@ -0,0 +1,312 @@
+"use strict"
+// GLOBAL VARIABLES
+const TEMP_DATA = {
+  hot: {
+    F: 80,
+    C: 27,
+    icon: '🐍  🦂 🌵 🌵 🐍  🏜  🦂 🌵'
+  },
+  warm: {
+    F: 70,
+    C: 21,
+    icon: '🌸🌿 🌼 🌷🌻🌿 ☘️🌱 🌻🌷 🌼'
+  },
+  mild: {
+    F: 60,
+    C: 15,
+    icon: '🌾🌾 🍃 🪨  🛤 🌾🌾🌾 🍃 🌾'
+  },
+  cold: {
+    F: 50,
+    C: 10,
+    icon: '🌲 🌲 🍂 🌲 🍁 🌲 🍂 🌲 🍂'
+  },
+  freeze: {
+    F: 40,
+    C: 5,
+    icon: '☃️ 🏂 ❄️ 🥶 ☃️ ❄️ ☃️ 🏂 ❄️'
+  }
+};
+
+const SKY_ICONS = {
+  Clear: '          ☀️           ',
+  Clouds:'☁️ ☁️ ☁️ ☁️ 🌤 ☁️ ☁️ ☁️ ☁️',
+  Rain: '🌧️ 🌧️ 🌧️ 🌧️ 🌧️ 🌧️ 🌧️ 🌧️ 🌧️',
+  Thunderstorm: '⛈️ ⛈️ ⛈️ ⛈️ ⛈️ ⛈️ ⛈️ ⛈️ ⛈️',
+  Snow: '🌨️ ❄️ 🌨️ ❄️ 🌨️ ❄️ 🌨️ ❄️ 🌨️',
+  Fog: '🌫️🌫️🌫️🌫️🌫️🌫️🌫️🌫️🌫️🌫️🌫️🌫️🌫️',
+};
+
+const WEATHER_IDS = {
+  'Clear': '800',
+  'Clouds': '8',
+  'Fog': '7',
+  'Snow': '6',
+  'Drizzle': '5',
+  'Rain': '3',
+  'Thunderstorm': '2'
+};
+
+
+const state = {
+  temp: null,
+  tempValue: null,
+  increaseTempButton: null,
+  decreaseTempButton: null,
+  currentTempButton: null,
+  tempFahrenheit: null,
+  tempCelsius: null,
+  cityNameHeader: null,
+  cityNameInput: null,
+  cityNameReset: null,
+  skySelect: null,
+  sky: null,
+  landscape: null,
+  gardenContent: null
+};
+
+const WEATHER_URL = 'https://weather-report-proxy-server-p10d.onrender.com/weather';
+const LOCATION_URL = 'https://weather-report-proxy-server-p10d.onrender.com/location';
+
+/////////////////
+// TEMPERATURE //
+/////////////////
+const getCurrentForecast = () => {
+  findLatAndLon(state.cityNameInput.value)
+    .then (response => {
+      return  getForcast(response.lat, response.lon)
+    })
+    .then((weatherData) => {
+      state.temp.textContent = convertKelvinToFahrenheit(weatherData[0]);
+      state.tempValue = parseInt(state.temp.textContent);
+      setColorAndLandscape();
+      setSkyByCityLookup(weatherData[1]);
+      toggleSelectedUnit(state.tempFahrenheit, state.tempCelsius);
```

Comment:
Could create a helper function that's descriptively named (maybe something like `updateDisplayForCurrentForecast`) and invoke these 3 methods there, which would make this `then` block a little more concise.

```suggestion
      updateDisplayForCurrentForecast(weatherData[1], state.tempFahrenheit, state.tempCelsius);
```